terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/resources/META-INF/functions.tld

```diff
@@ -109,32 +110,35 @@
             As a result, null value is not converted to a reset parameter that start with "_".
             In this case, this function does not guarantee symmetry with the form binding provided by Spring Web MVC.
         ]]>
-        </description>
-        <name>query</name>
-        <function-class>org.terasoluna.gfw.web.el.Functions</function-class>
-        <function-signature>java.lang.String query(java.lang.Object)</function-signature>
-        <example><![CDATA[
+    </description>
+    <name>query</name>
+    <function-class>org.terasoluna.gfw.web.el.Functions</function-class>
+    <function-signature>java.lang.String query(java.lang.Object)</function-signature>
+    <example><![CDATA[
 <a href="search?page=2&${f:query(searchForm)}">next page</a>
-        ]]></example>
-    </function>
+        ]]>
+    </example>

```

Comment:
- [ ] Please divide the commit between the part changed manually and the part changed by formatter.